docs: add new demo example for highlight menu

irma-kurnia-alaska opened the pull request:
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #169 

## Summary:

- Set the highlight menu option using `selected` attribute in auro-menuoptions

## Type of change:

- [x] Other (demo update) 


## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

irma-kurnia-alaska committed:
docs: add new demo example for highlight menu

Change to be committed:
newfiles: apiExamples/selected.html
modified: docs/partials/index.md

jordanjones243 commented on apiExamples/selected.html:
```diff
@@ -1,10 +1,6 @@
-</auro-input>
-<br />
-<auro-menu id="matchWordMenu">
+<auro-menu>
```
Why did you remove the `matchWord` example? The highlight menu option should be an additional example. Removing this example has also caused your build to fail.

irma-kurnia-alaska replied:
Hi @jordanjones243,

sorry I accidentally deleted that, i will back the example and add the example new for the highlight menu.

jordanjones243 reviewed (commented):
You need to run your build to actually build out the documentation file to be used on the website. Right now, if you ran the localhost you don't even see the example.

jordanjones243 reviewed (changes requested):
Accidentally submitted as a comment, but I am requesting changes.

irma-kurnia-alaska committed:
docs: revert the deleted example html

Change to be committed:
new file:  apiExamples/matchWord.html
new file:  apiExamples/matchWord.js
modified:  README.md
modified:  demo/index.md

irma-kurnia-alaska commented:
Hi @jordanjones243,

I will to rebuild docs now, thank you for your information.

jordanjones243 commented:
The work for the ticket attached to this will now be handled by this PR:
- https://github.com/AlaskaAirlines/auro-menu/pull/195
